core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -567,6 +568,19 @@ public SchemaString get(int id) throws SchemaRegistryException {
   }
 
   @Override
+  public boolean hasSubjectConfig(String subject) throws SchemaRegistryStoreException {
```

Comment:
As mentioned below, I think it's ok to remove this method and rely on the guarantee that schemaRegistry.getCompatibilityLevel(subject) returns null iff there is no subject-level override.

I guess that's to a map implementation that returns null when you try to fetch a key which isn't in the keySet